lib/locales/ru.yml

```diff
@@ -47,9 +47,9 @@ ru:
       color: [красный, зеленый, синий, желтый, багровый, мятный, зеленовато-голубой, белый, черный, оранжевый, розовый, серый, красно-коричневый, фиолетовый, бирюзовый, желто-коричневый, небесно голубой, оранжево-розовый, темно-фиолетовый, орхидный, оливковый, пурпурный, лимонный, кремовый, сине-фиолетовый, золотой, красно-пурпурный, голубой, лазурный, лиловый, серебряный]
       department: ["Книги", "Фильмы", "музыка", "игры", "Электроника", "компьютеры", "Дом", "садинструмент", "Бакалея", "здоровье", "красота", "Игрушки", "детское", "для малышей", "Одежда", "обувь", "украшения", "Спорт", "туризм", "Автомобильное", "промышленное"]
       product_name:
-        adjective: [Маленький, Эргономичный, Грубый, Интеллектуальный, Великолепный, Невероятный, Фантастический, Практчиный, Лоснящийся, Потрясающий]
-        material: [Стальной, Деревянный, Бетонный, Пластиковый, Хлопковый, Гранитный, Резиновый]
-        product: [Стул, Автомобиль, Компьютер, Берет, Кулон, Стол, Свитер, Ремень, Ботинок]
+        adjective: [Маленький, Эргономичный, Грубый, Интеллектуальный, Великолепный, Невероятный, Фантастический, Практчиный, Лоснящийся, Потрясающий, Огромный, Удовлетворительно, Синергетический, тяжелый, Легкий, Аэродинамический, Прочный]
```

Comment:
Two things here: The Russian terms are not all in their correct forms (e.g. new `material` entries are not all adjectives). Second is that like many languages, Russian assigns genders to inanimate objects. All `product`s have been masculine so far, but the new entries contain feminine items as well, which would clash with the all-masculine `material` entries. While the above things are simple fixes, they make me think that it would be good to similarly proof the `sv` locale.

In general, any thoughts on how we can deal with PRs that contain locale-specific data that neither of us can validate?